### PR TITLE
Remove unused labels from nodesets.yaml

### PR DESCRIPTION
**Pull request by lewisdenny**
With the bump to CRC 2.30 some labels are no longer used.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

**Comment by lewisdenny** [comment]
Hmm, this PR is failing because of [1], I guess this is why we don't use feature branches. 

https://github.com/infrawatch/service-telemetry-operator/blob/4f85c3ed7757bc9d6bc7f77f917d90931474626f/.zuul.yaml#L37

**Comment by lewisdenny** [comment]
The branch looks old so I requested the owner delete it if unused.

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
recheck

**Comment by lewisdenny** [comment]
Okay Zuul is still crazy so I'll try excluding all branches other than master for infrawatch/service-telemetry-operator

**Comment by lewisdenny** [comment]
https://review.rdoproject.org/r/c/config/+/51927

**Comment by lewisdenny** [comment]
recheck

File: zuul.d/nodeset.yaml [hunk 1]
```diff
@@ -53,17 +23,6 @@
       - name: peers
         nodes: []
 
-- nodeset:
-    name: multinode-centos-9-stream-crc
```
Comment (marios) [inline zuul.d/nodeset.yaml:57]: does this need a rebase? i cannot find it in https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/nodeset.yaml

Comment (marios) [reply zuul.d/nodeset.yaml:57]: yes seems removed with https://github.com/openstack-k8s-operators/ci-framework/commit/87e215703394f711e3f4cd95ebfd3e1000740d8e

**Review by marios — COMMENTED** [review]
lgtm but needs rebase?

**Review by marios — APPROVED** [review]
lgtm

**Comment by pablintino** [comment]
/approve
I have double checked if they are not needed and they are not. Thanks for cleaning the nodeset.

**Pull request merged**